### PR TITLE
Added Segment Install Steps

### PR DESCRIPTION
**Pull request by kelsric**

### CODE REVIEW

File: START.md [hunk 1]
```diff
@@ -202,6 +202,17 @@ You can use Taplytics to create [Experiments](https://taplytics.com/docs/android
...
+1. Log in to your [Segment Account](https://segment.com?utm_source=partners&utm_medium=Docs&utm_campaign=taplytics.com) 
+2. Navigate to the integration page for the project you want to install Taplytics.
+3. Scroll through the integration list and turn on the integration for Taplytics.
+    ![Image of URL Scheme](http://taplytics.com/assets/docs/install-sdk/segment-integration.png)
+4. You will now be prompted to add your Taplytics API key. Log in to your [Taplytics Account](https://taplytics.com/login/) and navigate to settings. Once there copy your Taplytics iOS/Android SDK Key.
+    ![Image of URL Scheme](http://taplytics.com/assets/docs/install-sdk/taplytics-sdkkey.png)
+5. Head back to Segment and paste in your key. 
+6. Press Save and Enable in the right-hand corner. 
+
+---
```
Comment (VicV) [inline START.md]: This is so much to scroll through before actually getting to the setup. :( cc @nemo

Comment (nemo) [inline START.md]: rm -rf Eclipse?

Comment (nemo) [inline START.md]: / it was always like that? lol.

Comment (VicV) [inline START.md]: No, these are big images. Check the actual page out.

Comment (nemo) [inline START.md]: We *could* put all the Segment stuff in its own file, too. @kelsric?

Comment (nemo) [inline START.md]: Would be interesting to keep it consistent across all the SDKs. Have a file called Segment.md and link to it from the START.md file?

**Comment by kelsric** [comment]
I am fine with whatever you guys want to do. It just needs to be visible so we can get added to the free tier. They are the same instructions for both SDK let me know how I can help

**Comment by kelsric** [comment]
Vic I changed my master does something like this work better ?

File: README.md [hunk 2]
```diff
@@ -10,8 +10,9 @@ _How do I, as a developer, start using Taplytics?_
 
 1. _Sign up for a free account at [Taplytics.com](https://taplytics.com?utm_source=github&utm_campaign=documentation&utm_medium=content)._
 2. _Install the SDK. Steps [here](/START.md)._
-3. Create [Experiments](/EXPERIMENTS.md) or send [Push Notifications](/PUSH.md) to your users!
+3. _Install the SDK Using Segment [here](/SEGMENT.md)_
```
Comment (VicV) [inline README.md]: No this seems like its a required step. Just put it in it's own 'SEGMENT' section again and just link to the file from there.

**Comment by kelsric** [comment]
@VicV very true check it out now I think this will work.

**Pull request merged**